CLIENT-SPECIFICATION.md

```diff
@@ -31,23 +31,23 @@ This section describes the standardised command-line interface (CLI) for clients
 
 ### Arguments
 
-A number of command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
+Several command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
 
 Option             | Required?   | Meaning
 -------------------|-------------|----------
 `-v`, `--version`  | Yes         | Shows the current version of the client, and the version of this specification that it implements.
 `-p`, `--platform` | Yes         | Specifies the platform to be used to perform the action (either listing or searching) as an argument. If this option is specified, the selected platform MUST be checked first instead of the current platform as described below.
-`-u`, `--update`   | Conditional | Updates the offline cache of pages. MUST be implemented if cache is supported.
+`-u`, `--update`   | Conditional | Updates the offline cache of pages. MUST be implemented if the cache is supported.
```

Comment:
Here if either go with "if a cache is supported" or "of caching is supported".

Reply:
```suggestion
`-u`, `--update`   | Conditional | Updates the offline cache of pages. MUST be implemented if caching is supported.
```